sources/packages/api/src/route-controllers/student/student.aest.controller.ts

```diff
@@ -0,0 +1,36 @@
+import { Controller, Get, Injectable, Param } from "@nestjs/common";
+import { ApiTags } from "@nestjs/swagger";
+import { AuthorizedParties } from "../../auth/authorized-parties.enum";
+import { AllowAuthorizedParty, Groups } from "../../auth/decorators";
+import { UserGroups } from "../../auth/user-groups.enum";
+import BaseController from "../BaseController";
+import { StudentUploadFileDTO } from "./models/student.dto";
+import { StudentControllerService } from "./student.controller.service";
+
+/**
+ * Student controller for AEST Client.
+ */
+@AllowAuthorizedParty(AuthorizedParties.aest)
+@Groups(UserGroups.AESTUser)
+@Controller("students")
+@ApiTags("students")
```

Comment:
We follow using API tags in a clienttype specific pattern. 
![image](https://user-images.githubusercontent.com/54600590/160000333-88c3906a-09d0-4837-a961-3ed6c2fb9a68.png)